fossunited/chapters/doctype/foss_event_rsvp_submission/foss_event_rsvp_submission.py

```
@@ -18,20 +18,24 @@ class FOSSEventRSVPSubmission(Document):
             FOSSCustomAnswer,
         )
 
         chapter: DF.Data | None
+        check_ins: DF.Table[EventCheckIn]
         confirm_attendance: DF.Check
         custom_answers: DF.Table[FOSSCustomAnswer]
         email: DF.Data
         event: DF.Data
         event_name: DF.Data | None
-        im_a: DF.Literal["", "Student", "Professional", "FOSS Enthusiast", "Other"]  # noqa: F722, F821
+        im_a: DF.Literal["", "Student", "Professional", "FOSS Enthusiast", "Other"]
```

Ruff (0.14.5)

Syntax error in forward annotation: Expected an expression

(F722)